cfn-resources/autogen/configs/mongodb-atlas-serverlessinstance-req.json

```diff
@@ -0,0 +1,46 @@
+{
+  "CreateFields": {
```

Comment:
What is the purpose/difference between resource.json and resource-req.json files?

Reply:
the req is a file generated by the automation, is used to store some metadata used in the code generation:
for example CreateFields.InputParams tags, contains potencial fields that might be needed to call the Create function in the atlas client. and the CreateFields.RequiredParams is used, to create a list of potencial required parameters we might need to validate if the model is completed.